packages/@lwc/wire-service/package.json

```diff
@@ -5,6 +5,7 @@
     "license": "MIT",
     "main": "dist/commonjs/es2017/wire.js",
     "module": "dist/modules/es2017/wire.js",
+    "typings": "lib/index.d.ts",
```

Comment:
is this something that was missing from before or is related to this functionality?

Reply:
missing from before... not related.